doc/guides/debugging/stack-size-analyzer.rst

```diff
@@ -0,0 +1,31 @@
+.. _stack_size_analyzer:
+
+Stack size analyzer
```

Comment:
Id drop the size term since if in the future someone wants to print more than just the size.

Reply:
Good point. Will do.